diff --git a/android/app/build.gradle b/android/app/build.gradle
index 6be86b8..6a683db 100644
--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -109,6 +109,12 @@ tasks.register("prepareAssets") {
 		copy {
 			from "${projRoot}/textures/base/pack" into "${assetsFolder}/textures/base/pack"
 		}
+		copy {
+			from "${projRoot}/clientmods" into "${assetsFolder}/clientmods"
+		}
+		copy {
+			from "${projRoot}/games" into "${assetsFolder}/games"
+		}
 		copy {
 			from "${localeFolder}" into "${assetsFolder}/locale"
 		}
